lib/ImageWorker.js

```diff
@@ -11,7 +11,7 @@ const webWorkerScript = `
         cache: 'default'
     }).then(response => {
         return response.blob();
-    }).then(_ => postMessage(url));
+    }).then(_ => postMessage(url)).catch(console.error);
```

Comment:
Just a thought.. do we need to log the error..or use postmessage in the failure block, and load the image normally? without the worker fetch

Reply:
That's a good idea, although most of the time this error means 404 etc, so the image won't be loaded anyway.

Reply:
we should log when its an error :)

Reply:
I think we are good to go :)

@kurtextrem good job :)